templates/starter-template/index.html

```diff
@@ -1,122 +1,157 @@
+    <div class="container">
+      <div class="section">
+        <div class="row">
+          <div class="col s12 m4">
+            <h2 class="center-align">
+              <i class="medium material-icons">flash_on</i>
+            </h2>
+            <h5 class="center-align">Speeds up development</h5>
+
+            <p>
+              We did most of the heavy lifting for you to provide a default
+              stylings that incorporate our custom components. Additionally, we
+              refined animations and transitions to provide a smoother
+              experience for developers.
+            </p>
           </div>
-        </div>
-
-        <div class="col s12 m4">
-          <div class="icon-block">
-            <h2 class="center light-blue-text"><i class="material-icons">group</i></h2>
-            <h5 class="center">User Experience Focused</h5>
 
-            <p class="light">By utilizing elements and principles of Material Design, we were able to create a framework that incorporates components and animations that provide more feedback to users. Additionally, a single underlying responsive system across all platforms allow for a more unified user experience.</p>
+          <div class="col s12 m4">
+            <h2 class="center-align">
+              <i class="medium material-icons">group</i>
+            </h2>
+            <h5 class="center-align">User Experience Focused</h5>
+
+            <p>
+              By utilizing elements and principles of Material Design, we were
+              able to create a framework that incorporates components and
+              animations that provide more feedback to users. Additionally, a
+              single underlying responsive system across all platforms allow for
+              a more unified user experience.
+            </p>
           </div>
-        </div>
-
-        <div class="col s12 m4">
-          <div class="icon-block">
-            <h2 class="center light-blue-text"><i class="material-icons">settings</i></h2>
-            <h5 class="center">Easy to work with</h5>
 
-            <p class="light">We have provided detailed documentation as well as specific code examples to help new users get started. We are also always open to feedback and can answer any questions a user may have about Materialize.</p>
+          <div class="col s12 m4">
+            <h2 class="center-align">
+              <i class="medium material-icons">settings</i>
+            </h2>
+            <h5 class="center-align">Easy to work with</h5>
+
+            <p>
+              We have provided detailed documentation as well as specific code
+              examples to help new users get started. We are also always open to
+              feedback and can answer any questions a user may have about
+              Materialize.
+            </p>
           </div>
         </div>
       </div>
-
+      <br /><br />
```

Comment:
```suggestion
      <br><br>
```

This may be better and is valid HTML5 =)

Reply:
Noted. let me know if you have more changes and I can do them all at once. I work on a lot of React stuff so sometimes my brain throws JSX or JS syntax into markup :P

Reply:
No problem, I know that too well (also working on React projects with TypeScript and PHP projects, I've tried to use the PHP function `echo` instead of `console.log` in a project a few days ago).